tests/functional/adapter/materialization/test_materialized_view.py

```diff
@@ -4,6 +4,7 @@
     check_relations_equal,
     run_dbt,
     run_sql_with_adapter,
+    run_dbt_and_capture,
```

Comment:
```suggestion
    run_dbt,
    run_dbt_and_capture,
    run_sql_with_adapter,
```
Please reorder imports alphabetically to satisfy the pre-commit check